initdb.py

```diff
@@ -93,7 +94,8 @@ def connect_to_database(connection):
     try:
         connection.database = DB_NAME
     except mysql.connector.Error as mysql_error:
-        raise Exception('There was a problem connecting to the database {}:\n{}'.format(DB_NAME, mysql_error))
+        raise MySqlError(message='There was a problem connecting to the database {}:\n{}'.format(DB_NAME, mysql_error),
```

Comment:
Don't put `mysql_error` in the message. The end-user doesn't need to see the stacktrace or original error. That's why we have `args`.